api/src/app.ts

```diff
@@ -19,5 +20,6 @@ app.use("/users", usersRouter);
 app.use("/games", gamesRouter);
 app.use("/connections", connectionRequestsRouter);
 app.use("/activity", userActivityLogRouter);
+app.use("/ranks", ranksRouter);
```

Comment:
Keep this for the / and /{id} endpoint, but all the rest will most likely move to the other top level endpoints (mainly `/users`)